va/va.h

```diff
@@ -1028,6 +1028,13 @@ typedef enum {
      * The value returned uses the VAConfigAttribValEncPerBlockControl type.
      */
     VAConfigAttribEncPerBlockControl    = 55,
+    /**
+     * \brief Maximum number of input streams for video composition in video processing. Read-only.
+     *
+     * This attribute determines the maximum number of input streams which the driver supports for
+     * a given configuration.
+     */
+    VAConfigAttribMaxInputStreams       = 56,
```

Comment:
1. suppose better to rename VAConfigAttribMaxInputStreams to VAConfigAttribVideoProcMaxInputStreams or VAConfigAttribMaxBatchSize , it is a VPP only attributes?
2. why not add a field into VAProcPipelineCaps? it is a VPP pipeline related parameters, max number of pipelines per submission . suppose it could be a part of pipeline parameter， it doe not conflict with multiple pipelines have different filter group